evennia/web/website/templates/website/_menu.html

```diff
@@ -40,8 +40,21 @@
             {% endblock %}
             {% block navbar_user %}
             {% if user.is_authenticated %}
-            <li class="nav-item">
-                <a class="nav-link">Logged in as {{user.username}}</a>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="user_options" aria-expanded="false">Logged in as {{user.username}} <span class="caret"></span></a>
+              <div class="dropdown-menu" aria-labelledby="user_options">
+                <a class="dropdown-item" href="{% url 'chargen' %}">Create</a>
+                <a class="dropdown-item" href="#">Manage</a>
+                <div class="dropdown-divider"></div>
+                {% for character in user.characters|slice:"10" %}
+                  <a class="dropdown-item" href="#">{{ character }}</a>
+                {% empty %} 
+                  <a class="dropdown-item" href="#">No characters found!</a>
```

Comment:
This section didn't work for me. When I logged in with my pre-existing superuser, my current Character was shown here, but creating a new character in the web chargen did not expand the list. I also tried registering a new Account, and even after having created a new Character (and played with it in the web client), this menu consistently showed "No characters found!". I have not examined why this is.

Reply:
Which multisession mode? That might be related.

Reply:
I think this probably was with mode 0.

Reply:
Well, obviously the goal is to do it *right*, but I was asking for the sake of diagnosis :)

Reply:
This was due to the use of lazy-property; the values were only being refreshed on server reload.